### PR TITLE
Improve documentation for Interpolated distribution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: pymc3/distributions/continuous.py [hunk 1]
```diff
@@ -1393,23 +1393,30 @@ def logp(self, value):
 
 class Interpolated(Continuous):
     R"""
-    Probability distribution defined as a linear interpolation of
-    of a set of points and values of probability density function
-    evaluated on them.
+i   Univarite probability distribution defined as a linear interpolation
```
Comment (junpenglao) [inline pymc3/distributions/continuous.py]: There is a mistake `i` here, also Univarite -> Univariate

File: pymc3/distributions/continuous.py [hunk 2]
```diff
@@ -1393,23 +1393,30 @@ def logp(self, value):
...
-    they are constant and cannot be sampled.
+    The lattice can be uneven, so the steps between different points can have
+    different size and it is possible to vary the precision between regions
+    of the support.
 
-    ========  =========================================
-    Support   :math:`x \in [x_points[0], x_points[-1]]`
-    ========  =========================================
+    The probability density function values don't have to be normalized, as the
+    interpolated density is anyway normalized to make the total probability be
```
Comment (junpenglao) [inline pymc3/distributions/continuous.py]: anyway -> any way
total probability be -> total probability

Comment (ghost) [reply pymc3/distributions/continuous.py]: @junpenglao Thank you! I updated the commit.

**Comment by junpenglao** [comment]
Thanks @a-rodin !

**Pull request merged**